### PR TITLE
Style guide for open source contributors

### PR DESCRIPTION
**Pull request by Whatthefoxsays**
# Fixes # 

### Checklist
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guide] and my PR follows them.
- [ ] I updated my branch with the master branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added in line documentation to the code I modified

### Short description of what this PR does:
- Added a style guide for open source contributors to follow. 
- 

If you have questions, please send an email to [Sendgrid](mailto:docs@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

File: styleguide.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+<a name="style"></a>
+## Style
+
+We strive to make our documentation concise, clear, organized, and scannable. 
+
+* Use conversational language and industry-standard terms when possible.
+* Contractions are OK and preferred.
+* Use bulleted lists and numbered steps where applicable. 
+
+When giving a user an action item, put the UI element involved in the action in **bold**. Avoid using the word "button" in the step and simply refer to the name on the UI element.
```
Comment (ksigler7) [inline styleguide.md]: This seems awkward, maybe: `When documenting a UI action, make the action **bold**`

File: styleguide.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+We strive to make our documentation concise, clear, organized, and scannable. 
+
+* Use conversational language and industry-standard terms when possible.
+* Contractions are OK and preferred.
+* Use bulleted lists and numbered steps where applicable. 
+
+When giving a user an action item, put the UI element involved in the action in **bold**. Avoid using the word "button" in the step and simply refer to the name on the UI element. 
+    For example, "Enter your username and password, and then click **login**." 
+
+Where possible, pages are written in a "jobs to be done" format. Jobs to be done style includes:
```
Comment (ksigler7) [inline styleguide.md]: Where possible, write pages in a "jobs to be done" format.

File: styleguide.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+{% endanchor h2 %}
+
+- [link text]({{root_url}}/User_Guide/Marketing_Campaigns/design_editor.html#-Using-Custom-HTML)(https://sendgrid.com/docs/User_Guide/Marketing_Campaigns/campaign_stats.html)
+- [link text]({{root_url}}/User_Guide/Marketing_Campaigns/design_editor.html#-Using-Custom-HTML)(https://sendgrid.com/docs/User_Guide/Marketing_Campaigns/campaign_stats.html)
+- [link text]({{root_url}}/User_Guide/Marketing_Campaigns/design_editor.html#-Using-Custom-HTML)(https://sendgrid.com/docs/User_Guide/Marketing_Campaigns/campaign_stats.html)
+```
+
+<a name="terminology"></a>
+## Terminology
+
```
Comment (ksigler7) [inline styleguide.md]: this list oculd be more consistent

File: styleguide.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+{% info %}
+Your note
+{% endinfo %}
+
+```
+
+
+<a name="screenshots"></a>
+## Screenshots
+
```
Comment (ksigler7) [inline styleguide.md]: extra line?

File: styleguide.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+Not every single page or step needs an accompanying screenshot. When in doubt, follow the guidelines below. 
+
+### Needs a screenshot:
+
+* The layout of the main elements and areas of the interface with captions. For example, the toolbar, status bar, main menu, pop-up menu.      This avoids needing to explain in words: “On the bar on the left, click on the second button from the top”.
```
Comment (ksigler7) [inline styleguide.md]: extra space between sentences

File: styleguide.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+* A screenshot for each action that significantly changes the state of the system, interface or data.
```
Comment (ksigler7) [inline styleguide.md]: ~~A screenshot for~~ **E**ach action that significantly changes the state of the system, interface or data.

File: styleguide.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+### Does NOT need a screenshot:
+
+* Install and uninstall wizards (unless there is something really unusual about a lot of the settings).
+* Login pages.
+* Standard dialog windows (open, save).
+* Endless drop-down lists (select language or country).
+* Individual buttons (use text).
+* Anything that is already described in detail or included on other screenshots – better to provide a link.
+* Screens containing details about the version, rights, etc. If a new version comes out, you may need to change numbers or letters on every    single screenshot.
```
Comment (ksigler7) [inline styleguide.md]: extra spaces

File: styleguide.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+* Pieces of code, configs, etc. Better to insert these as formatted text – it looks better, and users can easily copy them.
```
Comment (ksigler7) [inline styleguide.md]: Pieces of code, configs, etc. ~~Better to~~ **Instead,** insert these as formatted text – it looks better, and users can easily copy them.

File: styleguide.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+
+### File format and naming:
+
+Please use .png, .gif, or .jpg files. Give the screenshots unique, meaningful names, obeying a specific format, such as general_settings_privacy. Avoid spaces, uppercase letters,and excessively long names.
```
Comment (ksigler7) [inline styleguide.md]: missing space after letters

**Comment by thinkingserious** [comment]
Hello @Whatthefoxsays,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**